UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandlerManager.cs

```diff
@@ -246,6 +248,12 @@ public void MergeInto(SpriteChange spriteChange, bool pool = false)
 				PushClear = spriteChange.PushClear;
 			}
 
+			if (spriteChange.AnimateOnce)
+			{
+				if (AnimateOnce) AnimateOnce = false;
```

Comment:
so, this is just for  late clients, idk how should be handled to get the correct state but if I think if push texture has been applied Then cancel AnimateOnce it's a little bit more complex this bit

Reply:
note there may be more times when other stuff overrides it